cypress/integration/Transfer/disabled-transfer-buttons.feature

```diff
@@ -0,0 +1,42 @@
+@component-transfer @button-states
+Feature: Disable transfer buttons when actions are not possible
+
+    Scenario: None of the available options is highlighted
```

Comment:
None of the available options is highlighted -> None of the available options are highlighted